vms/volunteer/templates/volunteer/profile.html

```diff
@@ -35,13 +35,13 @@ <h4 class="panel-title">
                 <div class="panel-body">
                 <table>
                     <!-- Better to have css padding instead of nbsp to insert spaces -->
-                   <tr> <td><label><h4><span class="label label-info">Email</span></h4></label></td>&nbsp;&nbsp;<td align="center">{{ volunteer.email }}</td></tr>
-                    <tr><td><label><h4><span class="label label-info">City</span></h4></label></td>&nbsp;&nbsp;<td align="center">{{ volunteer.city }}</td></tr>
-                    <tr><td><label><h4><span class="label label-info">State/Province</span></h4></label></td>&nbsp;&nbsp;<td align="center">{{ volunteer.state }}</td></tr>
-                    <tr><td><label><h4><span class="label label-info">Country</span></h4></label></td>&nbsp;&nbsp;<td align="center">{{ volunteer.country }}</td></tr>
-                    <tr><td><label><h4><span class="label label-info">Shift Reminder Days</span></h4></label></td>&nbsp;&nbsp;<td align="center">{{ volunteer.reminder_days }}</td></tr>
+                   <tr> <td width=250px><label><h4><span class="label label-info">Email</span></h4></label></td><td align="left">{{ volunteer.email }}</td></tr>

```

Comment:
Pixels (`px`) are the absolute size. Could you change it to `%` from `px`?

Reply:
@tapasweni-pathak Yes % must be used, I have changed it. Please review.